Makefile

```diff
@@ -24,8 +24,10 @@ dev.node2:
 	REGION=eu \
 	SECRET_KEY_BASE="dev" \
 	CLUSTER_POSTGRES="true" \
+	PROXY_PORT_SESSION="5442" \
+	PROXY_PORT_TRANSACTION="6553" \
```

Comment:
Notice this env var is already set earlier.

Reply:
oops